api/src/main/java/org/pmiops/workbench/config/WorkbenchConfig.java

```diff
@@ -307,8 +307,6 @@ public static class FeatureFlagsConfig {
     // If true, copy the support staff when sending Admin Locking emails.
     public boolean ccSupportWhenAdminLocking;
     // If true, enable Multiple Reviews for Cohort Review - both backend and UI
-    public boolean enableDrugWildcardSearch;
-    // If true, enable Drug Wildcard Search for Cohort Builder - backend
```

Comment:
Story only says to remove references to flag, but not the flag itself. Please revert.